cmake/Dependencies.cmake

```diff
@@ -330,16 +330,27 @@ if(BUILD_TEST)
 
   # For gtest, we will simply embed it into our test binaries, so we won't
   # need to install it.
-  set(BUILD_GTEST ON CACHE BOOL "Build gtest" FORCE)
   set(INSTALL_GTEST OFF CACHE BOOL "Install gtest." FORCE)
-  # We currently don't need gmock right now.
-  set(BUILD_GMOCK OFF CACHE BOOL "Build gmock." FORCE)
+  set(BUILD_GMOCK ON CACHE BOOL "Build gmock." FORCE)
```

Comment:
Do we have to set `INSTALL_GMOCK` OFF here?

Reply:
no, the variable is still called `INSTALL_GTEST`, see https://github.com/google/googletest/blob/2fe3bd994b3189899d93f1d5a881e725e046fdc2/CMakeLists.txt